src/browser-frontend/views/browser/chrome/tabbar/tab/tab-favicon.jsx

```diff
@@ -36,15 +34,31 @@ export default class TabFavicon extends PureComponent {
 
     if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.INITIAL ||
         this.props.loadState === DomainPageMetaModel.LOAD_STATES.CONNECTING) {
-      contents = <SpinnerType1 />;
+      contents = (<FittedImage
+        src="var(--theme-tab-connecting)"
+        width="28px"
+        height="16px"
+        mode="contain"
+        className="spinner"
+        style={{ marginRight: '2px' }}
+      />);
     } else if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADING) {
-      contents = <SpinnerType2 />;
-    } else if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADED) {
+      contents = (<FittedImage
+        src="var(--theme-tab-loading)"
+        width="28px"
+        height="16px"
+        mode="contain"
+        className="spinner"
+        style={{ marginRight: '2px' }}
+      />);
+    } else if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADED ||
+              this.props.loadState === DomainPageMetaModel.LOAD_STATES.ANIMATED) {
       if (this.props.favicon !== DomainPageMetaModel.UNKNOWN_FAVICON) {
         contents = (<FittedImage
           src={`url(${this.props.favicon})`}
           width="16px"
           height="16px"
+          style={{ margin: '0 8px 0 6px' }}
```

Comment:
Are these extra margins here intentional? Looks a bit worse to my eye, but maybe this is the new spec!